### DISCUSSION TITLE
Integrating Syncfusion

### DISCUSSION THREAD

**Discussion by markagama** [oqtane/oqtane.framework#1719] [Q&A]
I was following the guide by @ADefWebserver found here https://blazorhelpwebsite.com/ViewBlogPost/47
The first thing I noticed is the code created by the current Oqtane is quite different from the example, but I moved ahead hoping there were no show stoppers.

And got up to the step
Using IHostResources to Include References in _Host.cshtml
I did not want to rework the previous example so I commented out the line
new Resource {ResourceType = ResourceType.Script,Url = "Modules/"+ GetType().Namespace+ "/CustomJavaScript.js"},
I then moved on to the next step
Consume Syncfusion In The Module
unfortunately the two files shown in the image don't exist in the code created by Oqtane
Index.razor and Edit.razor

I thought I would look at the earlier example found here https://blazorhelpwebsite.com/ViewBlogPost/45
to see if I could find where these files were created and I immediately noticed that in the section
Update the Code
it mentions files Module.js and Module.css in oqtane.server but the files don't exist in the current oqtane build.

At this point, I figure I'm stuck as there are changes in both examples from the current oqtane code.

Michael, if you're reading this, please help. I believe your guides are great and would be great in a FAQ, so they can be updated with each oqtane source code build.

**Comment by markagama** [comment]
Hi Again, I think that the Index.razor and Edit.razor have moved in the client to
Modules/the.module/

I'm going to proceed with this until I hear otherwise.

Michael, if the files have been moved, it would be great for your wonderful examples to be updated.

**Reply by ADefWebserver** [reply]
> Michael, if the files have been moved, it would be great for your wonderful examples to be updated.

Unfortunately 50% of all the screen shots would also have to be redone :( 
The tutorial is best used a a learning exercise against the [exact version of Oqtane it was written for](https://github.com/oqtane/oqtane.framework/releases/tag/v2.0.1), not the latest version.

**Reply by markagama** [reply]
Hi Michael, the examples are really great work, and I appreciate the help. I think I should be able to finish a new module and test based on the example.
